apps/wpcom-block-editor/src/wpcom/features/tracking/wpcom-site-editor-document-actions-dropdown-click.js

```diff
@@ -0,0 +1,66 @@
+import tracksRecordEvent from './track-record-event';
+
+/**
+ * Return the event definition object to track `wpcom-site-editor-document-actions-dropdown-clicks`.
+ *
+ * @returns {import('./types').DelegateEventHandler} event object definition.
+ */
+export const wpcomSiteEditorDocumentActionsDropdownOpen = () => ( {
+	id: 'wpcom-site-editor-document-actions-dropdown-open',
+	selector: '.components-dropdown .edit-site-document-actions__get-info',
+	type: 'click',
+	handler: () => tracksRecordEvent( 'wpcom_site_editor_document_actions_dropdown_open' ),
+} );
+
+/**
+ * Return the event definition object to track `wpcom-site-editor-document-actions-template-area-click`.
+ *
+ * @returns {import('./types').DelegateEventHandler} event object definition.
+ */
+export const wpcomSiteEditorDocumentActionsTemplateAreaClick = () => ( {
+	id: 'wpcom-site-editor-document-actions-template-area-click',
+	selector: '.components-dropdown__content .edit-site-template-details__template-areas',
+	type: 'click',
+	handler: ( event ) => {
+		const headerSVGSnippet =
+			'M18.5 10.5H10v8h8a.5.5 0 00.5-.5v-7.5zm-10 0h-3V18a.5.5 0 00.5.5h2.5v-8zM6';
+		const footerSVGSnippet =
+			'M18 5.5h-8v8h8.5V6a.5.5 0 00-.5-.5zm-9.5 8h-3V6a.5.5 0 01.5-.5h2.5v8zM6';
+
+		// There are no selectors that provide a distinction between the Header and Footer template
+		// area buttons. We could use text content instead, but that information is unreliable when
+		// the text is translated. Because of this, we look for accompanying SVG icons and derive
+		// information from that instead.
+
+		// The buttons have an icon AND text content. The user may click on either element. We check
+		// for both possibilities here.
+		const targetHTML = event.target.innerHTML;
+		const previousElementSiblingHTML = event.target?.previousElementSibling?.innerHTML;
+
+		if (
+			previousElementSiblingHTML?.includes( headerSVGSnippet ) ||
+			targetHTML?.includes( headerSVGSnippet )
```

Comment:
Curious if these cover the case of the ellipsis menus on the right of these buttons?
![Screen Shot 2022-11-02 at 8 55 47 AM](https://user-images.githubusercontent.com/28742426/199495069-ea754634-e522-4f33-86e8-f304e06b4797.png)

Reply:
The content shown in the ellipsis menus are captured by a different tracking event `wpcom_block_editor_nav_sidebar_item_edit`, but the ellipses themselves aren't. If we'd like that captured as well, I can make the changes.

Reply:
> content shown in the ellipsis menus are captured by a different tracking event wpcom_block_editor_nav_sidebar_item_edit

Hah, interesting! That makes sense given how thats tracked and I dont think is a huge issue.

> but the ellipses themselves aren't. If we'd like that captured as well, I'm happy to make the changes.

Yeah that would be great! Then when @annezazu and others are looking at data to help inform the redesign they will be able to get some idea about how often this section is used.

Reply:
On it -- Thanks for the feedback Addie!

Reply:
[236af1d](https://github.com/Automattic/wp-calypso/pull/69642/commits/236af1d314406bab0c9604ec7e21a6df5c8190b7)

Reply:
I also noticed there was a revert customizations button that appears when changes are made to template parts. I track included logic to track that as well @Addison-Stavlo